### ISSUE TITLE
Index doesn't display with double tap [In5?]

### ISSUE THREAD

**Issue by ghost** [Simbul/baker#1167]
Hi there,

Since upgrading to iOS 7 I am experiencing a bug with the index navigation bar.

As soon as the issue is launched on my iPad the index.html is visible and showing. 
However once its tapped away, it doesn't reappear. 

I'm wondering if this is an Baker or In5 issue as my current workflow is: Adobe InDesign > In5 > baker

[Please note this issue can also be found here - #1154]

**Comment by folletto** [comment]
#1154 is a different issue. ;)

That said, it seems a In5 issue. Did you try contact them and point them to this thread? :)

Also, please read through [Problems and Debugging](https://github.com/Simbul/baker/wiki/Problems-and-Debugging) and report back. ;)

**Comment by ghost** [comment]
@folletto Yep I just posted the question on their Answer Exchange [http://bit.ly/15VLji5]

I will go through the <a href="https://github.com/Simbul/baker/wiki/Problems-and-Debugging"> Problems and Debugging</a> section and get back to you.

[on a side note this might also explain why I seem to be the only one getting such crappy performance with my latest issue on iOS 7 iPad Mini]

**Comment by limster1** [comment]
Hi Thanks to @lukeallen1986 for bringing up this issue. I am also facing the same problem since upgrading my iPad to IOS7. We are using Laker Compendium to develop our AppBooks. Hope the update will happen soon. Thanks much.

**Comment by nicholasmartin** [comment]
Hmm in Laker as well?

I'll try and get a simple HTML5 test up. Thought this issue only pertained to in5 export feature.

**Comment by limster1** [comment]
Hi thanks for your quick response. Yes laker too. Appreciate yr help very much. Best wishes lim

**Comment by nicholasmartin** [comment]
I found the problem. I was testing in an older version of Baker. I just tested the in5 export in the iOS7 branch and the index double tap works as it should. However, when I tested in the older "preview" app that I had installed, the index file behaved as mentioned in this thread.

Try it out in iOS7 branch and see it works. Not sure if the master branch is updated yet.

**Comment by folletto** [comment]
The `iOS7` branch has been merged to `master`, use master now. :)

**Comment by nicholasmartin** [comment]
Awesome :)

**Comment by limster1** [comment]
Dear davide tq so much!! Much appreciate. So we download the latest baker master n it should work?  Thx again.

**Comment by limster1** [comment]
Dear Davide,

We have downloaded the new master with the ios7 branch merged. However, we
have encountered one error:

fatal error: file
'/Users/Wann/Downloads/baker-master/Baker/Baker-Prefix.pch' has been
modified since the precompiled header
'/Users/Wann/Library/Developer/Xcode/DerivedData/Baker-ajffqbssbzsitngmrocrvsbmdeqz/Build/Intermediates/PrecompiledHeaders/Baker-Prefix-gunnchljvohmznbxyhtuhctknsgh/Baker-Prefix.pch.pch'
was built

1 error generated.

Any idea how we can solve this? We have solved all the other errors.

Thanks
Lim

On Tue, Oct 8, 2013 at 11:13 PM, Davide Casali notifications@github.comwrote:

> The iOS7 branch has been merged to master, use master now. :)
> 
> —
> Reply to this email directly or view it on GitHubhttps://github.com/Simbul/baker/issues/1167#issuecomment-25898794
> .

## 

(__/)
(• o •)
(_/|_)
(=)_(=)

**Comment by folletto** [comment]
Please open a separate issue. This is about the index problem. ;)
_(seems a cleanup issue tho, follow Problems and Debugging from the wiki...)_
